### ISSUE TITLE
Kor automata ships should require a Command center and Bunks to be usable

### ISSUE THREAD

**Issue by starscollide5** [endless-sky/endless-sky#5059]
**Is your feature request related to a problem? Please describe.**
Kor automata ships are easy to obtain and extremely powerful. For some obscure reason, they do not require any crew members for use after capture, which is bad both for balance - player doesn't need to pay crew salaries for using any amount of them - and lore - player secretly being some kind of ultra-talented hacker able to easily hijack automata on a whim doesn't make any sense and goes against the whole Wanderer storyline. 

**Describe the solution you'd like**
I offer the following measures to mitigate this problem:
1. All weapons of any Kor automata ship under player control except drones should remain inactive unless a Command center outfit is installed on the automata ship. 
2. Attempting to capture an automata ship should consume a few (e.g. 2) crew members regardless of the result. 

Important note: as long as a Command center is installed, turret crew requirements already work for Kor automata ships the very same way they do for any other ship in the game. 

I believe it goes well with the lore - while a few crew members should be enough to hack an automata navigation system and get the engines going, weapon systems should not function until the ship is properly refitted for human control. 2 also makes it harder to capture many Kor automata ships in one go, bringing it more in line with capturing other ships in the game. 

**Additional context** (DROPPED)
~~Additional proposed change that  requires further discussion:~~
~~3. Add a 50% chance of automata ship attacking player after successful capture.~~

~~This change adds flavor, and makes capturing Kor automata ships a bit harder and riskier, particularly in a very weak ship completely incapable of holding their own against powerful automata.~~

**Comment by Zitchas** [comment]
I'm dubious about the automata going wild after capture unless that is a replacement for the self-destruct. If, intstead of the self-destruct we had the possibility of having the ship return to life at the current self-destruct odds... That'd be good.

**Comment by starscollide5** [comment]
@Zitchas I agree. After all, if automata comes back to life, player will need to disable it and pass the self-destruct check again.

**Comment by dragonmilk** [comment]
As an AEE cloaker in Mesuket, I already feel like it's already very grindy. I can't imagine any other ship being able to experience the 4% chance to build up large fleets of automata.

At that point, you're just adding barriers that may be more tedious than fun

**Comment by starscollide5** [comment]
@dragonmilk 

> At that point, you're just adding barriers that may be more tedious than fun

My main proposal is to require Control center (and bunk rooms for manned turrets) for Kor ships, do you think it's a bad proposal and makes the game more tedios? After capturing a Kor ship you'll likely want to tow it somewhere for repairs, or at least send it over to the edge of the map, and pray that it doesn't get blown up by odd automata fleet jumping in, not try to use it in battle immediately - so it doesn't change much in terms of gameplay but makes a lot of sense in terms of lore - or am I wrong in your opinion?

> As an AEE cloaker in Mesuket, I already feel like it's already very grindy.

I personally feel capturing Kor ships is very easy as things are. I have absolutely no problem doing it in a solo Splinter without cloaking device; it is perfectly doable in other small yet fast ships like Raven or Headhunter, as long as they are properly outfitted. 

>  I can't imagine any other ship being able to experience the 4% chance to build up large fleets of automata.

What about Korath Raider, Quarg ships, and Korath World-Ship that is currently impossible to capture?

**Comment by gunqqer** [comment]
Capturing Kor ships is an intended features. Raiders, Quarg, and World-Ships are not supposed to be capturable. 

As for how easy capturing Kor ships, well, I guess you could call it easy as it requires no thought, but the problem is how long it takes especially if you want to get a 349 or 512. Anecdotaly I have spent hours trying to get just a few of them. That is hard (on the psyche).

**Comment by dragonmilk** [comment]
Without cloak, you actually have to:
1. Actively make sure your ship isn't dying
2. Avoid aggro by getting too close to a battle
3. Carefully navigate the systems to and from a planet with a spaceport.

If you have hours to spare, be my guest, but unless you have an AEE, you're missing the efficient process of:

1. Cloak into the system
2. Press caps lock to fast forward
3. Eat lunch (or do whatever for a while)
4. Come back and salvage spread out carcasses one at a time with brief moments of decloak

And even so, it's a very tedious process.

**Comment by starscollide5** [comment]
@gunqqer shouldn't powerful ships be hard to acquire, then? 

I'll make a few other examples:
* A pirate Bactrian (or a civilian one for that matter) can be quite hard to find and capture unless player progresses over FW or Deep storylines
* most Marauder heavy warships require high combat rating, although there is one storyline
* powerful Wanderer ships are not available unless player makes a lot of progress in Wanderer storyline
* Pug Arfecta is also quite hard to capture
* Pug Maboro
* Heliarch Punisher

Third proposal is indeed very controversial, I wanted to see feedback on this idea, and so far I'm getting it. However, seeing as it drags all focus from first two, it seems I should have created a separate issue for it... 

@dragonmilk without cloaking device, you just park your ship far from the system center, turn on caps lock and leave Endless Sky be while disabled Kor ships pile up in Mesuket. When there are quite enough of those, you approach the ones you're interested in that are farther from ongoing battle, and try to capture them. When you manage to capture one, it is sensible to either leave the system, or at least send it to safety before attempting to capture another one, or it other automata might get interested in it. 

I don't understand why would you need a cloaking device for this. Trying to capture disabled Kor automata near the ongoing fight might result in your freshly captured ship being immediately disabled, and will likely produce tons of frustration, won't it?

**Comment by dragonmilk** [comment]
A cloaking device drops aggro. If you approach a ship without a cloaking device, you may risk getting you or your target destroyed - with a cloaking device, you can board even if aggroed and zip out. Otherwise you have to be much more cautious. 

Part of your actual proposal doesn't make ships any harder to capture as I read it, it's merely adding tedium of dropping by an outfitter to buy a command center before you make them actually operational. The other part of expending 2 crew per attempt just adds to tedium - having to jump 5 systems to restock on crew before pressing caps lock again to wait for disabled ships between failed attempts doesn't really make this game any more fun to play.

AEE is an Arfecta

**Comment by starscollide5** [comment]
> A cloaking device drops aggro. If you approach a ship without a cloaking device, you may risk getting you or your target destroyed - with a cloaking device, you can board even if aggroed and zip out

I'd argue with a fast (900+, preferably over 1000) and agile (90+ turn rate) ship you only need be slightly more cautious than with a cloaked one... but I admit that's opinion-based, and different playstyles require different approaches. 

> Part of your actual proposal doesn't make ships any harder to capture as I read it, it's merely adding tedium of dropping by an outfitter to buy a command center before you make them actually operational. The other part of expending 2 crew per attempt just adds to tedium - having to jump 5 systems to restock on crew before pressing caps lock again to wait for disabled ships between failed attempts doesn't really make this game any more fun to play.

After capturing most powerful ships, e.g. Carrier, Hurricane or Punisher even on a Bactrian with maxed out bunks player will have  dock and replenish crew before capturing a next one. Why should should it be any different for equally powerful automata ships? Admittedly player gets annoying self-destruct chance instead of having to actually fight the defenders, but this actually makes capturing easier and less tedious when compared to ships I mentioned. 

I in all honestly believe my first two proposals merely put capturing powerful automata ships more in line with capturing comparable ships. As things are, a Headhunter with atomics and a jump drive + some patience get you a fleet of extremely powerful ships without spending a single crew member on capturing them, a single credit on crew salary, or ever needing to dock to replenish crew after capturing any amount of them. I believe this is wrong and broken; while this ships should be capturable, they also should require some effort. 

> all that being said, I wouldn't be opposed to making kor ships capturable only if you or an escort personally disabled it - in which case the chance for capture necessarily should be much higher than the current 20% * 20% etc. Perhaps make it into 100% you can board, and [X%] based on ship type you can capture if you personally disabled the ship. I do think it's kinda lame you can be a vulture in Mesuket.

You can be a vulture anywhere in the galaxy, and capture any ship you didn't disable - why should it be different for Kor ships? They're just easier to capture than comparable ships, that's it.

**Comment by dragonmilk** [comment]
So you've basically proposed that capturing Korath ships be an extended tedious vulture exercise rather than based on actually disabling ships yourself like most of the game. I'll pass

**Comment by starscollide5** [comment]
> rather than based on actually disabling ships yourself like most of the game.

There are no restrictions on capturing ships player disabled anywhere in the game. Player can board and try to capture any ship disabled by anyone.  

> So you've basically proposed that capturing Korath ships be an extended tedious vulture exercise

I propose making capturing powerful Kor ships about as tedious as capturing other powerful ships in the game, nothing more and nothing less.

**Comment by Anarchist2** [comment]
How does "consuming a few crew members per boarding attempt" meaningfully increase the difficulty of boarding automata? It can be trivially bypassed by making a save somewhere like Spera Anatrusk, then loading a save from there whenever you run out of crew.

**Comment by starscollide5** [comment]
@Anarchist2 I am not trying to make capturing Kor ships impossible, merely bring it closer to capturing other ships. E.g. player on a Bactrian can afford to capture a few Shield Beetles before needing to land and replenish crew, while player on a bunked out Corvette can barely capture one. Why should it be any different for Kor ships?

**Comment by Anarchist2** [comment]
Even in the context of bringing it in line with standard capping, I don't think that it does enough in that regard.
Using the Shield Beetle comparison - A Centipede can cap a Shield beetle, but something like a clipper can't.
With automata, even with consuming crew members per cap attempt, pretty much every ship can cap one. All that changes is how often you have to reload a save/restock crew at an inhabited planet.

**Comment by starscollide5** [comment]
> Even in the context of bringing it in line with standard capping, I don't think that it does enough in that regard.

It improves the current situation and, in my opinion, fills the blanks in game lore, explaining how exactly is player able to capture and control the automata. Player being able to just get a full control of very sophisticated war machines without doing a major overhaul of their systems makes zero sense to me, if player has this know how they could certainly offer their expertise to assist Wanderers instead of running errands for them

> With automata, even with consuming crew members per cap attempt, pretty much every ship can cap one. All that changes is how often you have to reload a save/restock crew at an inhabited planet.

Just like a bunked out Clipper can capture a Shield Beetle by reloading save file enough times.

**Comment by beccabunny** [comment]
> Just like a bunked out Clipper can capture a Shield Beetle by reloading save file enough times.

...you just said that, given enough attempts and a proper outfitting, any ship can capture even shield beetles. How is that different from what happens with the Kor ships, where you have to reload many, many times and outfit a ship (for thrust+turn or cloak+fuel) to be able to survive and cap some?

**Comment by starscollide5** [comment]
@beccabunny it doesn't differ in principle, and why should it? I merely offer to make capturing Kor ships slightly harder and make their use understandable from lore point by reusing a few outfits already existing in the game.

**Comment by dragonmilk** [comment]
If no one agrees with your proposal, you might consider a mod rather than force your idea on everyone.

**Comment by starscollide5** [comment]
> If no one agrees with your proposal, you might consider a mod rather than force your idea on everyone.

Nah, I'll just close the issue then. 

Honestly I never had any problem with this until seeing a post on Reddit mocking player's ability to capture and control automata as one of the game's major inconsistencies from lore standpoint. After giving it some thought, player being able to easily control the automata indeed doesn't make much sense, hence this issue and proposed ways of overcoming it using mild changes and outfits already present in the game. 

Bottom line, if nobody wants it, I can certainly live without it. Though I believe it would make the game better.

**Comment by dragonmilk** [comment]
I didn't mean to come across as totally dismissive, but I definitely had a visceral reaction to increasing the annoyance of getting these ships. I've come back to 30+ 349s disabled and all have blown up, and imagining have to restock on crew as well is tilting, as it limits the amount you can board.

Anyway, I'm all for having to disable the ship yourself, as I mentioned, but since you were opposed to that and started justifying vulturehood, it seemed like you liked that anyone can pick it up, you just wanted to make that part harder, which I disagree with.

**Comment by starscollide5** [comment]
@dragonmilk Endless Sky caters to different playstyles. @Zitchas , me and many others mostly play solo or in small fleets. As of now, all missions present in the game are perfectly doable in a single ship, moreover they are doable in a single human ship with human outfits - although the latter definitely raises the bar of game difficulty. 

If your proposal was implemented, one odd shot by an NPC could destroy all the hard work a solo player puts into capturing enemy ship. And how can it be explained from lore standpoint, given that pretty much all factions except Pirates and Remnant shoot to kill instead of looting the enemy ships? It would just look like a brutal restriction completely out of place in a sandbox game, in my opinion. 

I also used to amass giant automata fleets, back in the day. I understand how hard and annoying it is as things already are. But on the other hand, amassing a fleet of Heliarch Punishers seems even harder to me. Even just disabling tens of them to steal some Finisher torpedoes that aren't available for purchase anywhere was pretty hard and took quite some time, while capturing is noticeably harder and requires landing after capturing one. For reference, while Punisher is tankier and agile, 349 is capable of packing a lot more punch when properly outfitted, so I dare say overall they are about the same tier. 

I dare say most Endless Sky playthroughs do not involve grinding hundreds of automata ships. Regular player on a pretty much any Large Warship, or many Medium Warships, wouldn't feel a thing if they had to sacrifice measly two crew members each time they attempt to board disabled automata leftovers from storyline battles, and refitting a few automata is not the same as refitting hundreds of them - all while filling the hole in lore about player's ability to capture them.

Anyway, if we go with the whole _player must disable an automata themselves_ business, how about this option - player boards a disabled automata and faces a self-destruct check like usual, then if it was player or their personal fleet who disabled the automata, player can proceed to capture it without additional self-destruct checks, while if it someone else landed the final shot, player gets a large chance of automata coming back to life and attacking the player on capture attempt?

**Comment by vitalchip** [comment]
@starscollide5, A plugin might be a good option. I'm in the 'it's too easy' and 'inconsistent' camp. For anyone who's played through more than once you know the easy route to getting stuff, so we either pretend they don't exist or look at ways to change the balance.

 Just to throw an idea into the mix:  I altered the Kor Sestor 349 ship so that once captured it would act as an Automaton escort but it was impossible to change the weapons without putting a crew on board. In other words, you captured a radio control ship and that's what you got, That Ship.

If I wanted to put (example) Skylances on it I have to take out the automaton control centre and put the required crew on board. I set it at 25, low for a ship this big, but let's assume it's still quite well automated. You no longer need a command centre, just remove the Automation Centre and put in bunks.

In my view this does away with the nonsense of installing all sort of other weapons and somehow the Sestor ship can control them, even though they were designed to be crewed eg Dual Sunbeam Turret needs 2 crew but bung it on a Sestor ship and it's able to make it work with zero.  

No weapon can be removed without first removing the Control Centre, the Control Centre cannot be installed again until all of the weapons are put back ( technically they can be swapped about for other weapons obtained from Sestor ships, but not any others ) The Control Centre cannot be installed in any other ship (  So no using it to make an Automaton version of a Punisher.. )

I think this is a reasonable compromise, most people would want to modify the ship and so they are pushed down the crew path. Some may just want to use it as the Automaton they captured and that's fine too. Plus of course if it's a plugin then it's a choice whether to use it or not.

Anyway, just an idea I thought I'd share. It seems to be a pretty divisive subject!  

p.s Zitchas is adding operating costs to these ships soon, a plugin like this wouldn't be any issue, easily balanced by playing around with required crew numbers and you'd still get a ship that's cheaper to run than anything comparable.

**Comment by geojak** [comment]
The friendly korsth exiles should sell some very expensive hacking outfits.

We reduce the explosion chance on boarding, but increase the self destruct chance to 1 when trying to capture without the hacking outfit. With the hacking outfit their is a chance higher than currently but the outfit is consumed on the attempt.

Capturing Bigger drone ships consume more hacking outfitts. 

In this game money comes easy and a giant fleet of wnsderes ships is doable. The hacking cost should be comparable to buying new wanderer ships.

Overall this should make capturing drones less of a grundy unfun pain compensated by needing more money.

**Comment by geojak** [comment]
Also haven't we discussed this year's ago already?

**Comment by gggainzzz** [comment]
if you leave the system it ALL resets and all those carcasses would be gone.  FUN times.  I don't agree it's easy.  It takes hours as some have already said so what is stopping you from ammassing a large fleet is time LOTs of it.  Obviously you could sit idle and wait hours [eat lunch] come back and die to a mass fleet jumping in on you and that speedy boy will die quick. wasting time and only creating more frustration.  I actually like fighting and disabling them. but again lots of work that makes most people decide to eat lunch which is fine but means it is seen as mostly a way to avoid tedium.

**Comment by starscollide5** [comment]
> if you leave the system it ALL resets and all those carcasses would be gone. FUN times.

The same happens with ships of other races. Player leaves or lands - they disappear, unless they belong to some mission. Player can't capture numerous large ships of other races because they require crew to both capture and pilot. I merely suggest putting Kor more in line with other races in this regard.

**Comment by tux2603** [comment]
One thing that you might do is only be able to capture the automaton if you have a certain outfit installed, possibly either some sort of computer that can perform a brute-force attack or a control key granted by/stolen from the Kor

**Comment by dragonmilk** [comment]
@tux2603 That's something I could totally be ok with - like some software unlocked through a mission quest (like mid-wanderer post *spoiler* on investigating *spoilers*)

**Comment by Amazinite** [comment]
> only be able to capture the automaton if you have a certain outfit installed

Ref: #2948

**Comment by Zatishko** [comment]
This problem goes away if one can install certain outfits in cargo while boarded, although Amazonite's Ref to #2948 is probably better. Nor does an empty outfitter in system, but out of the way, fix such things.

**Comment by Amazinite** [comment]
See the last two comments.
